### ISSUE TITLE
Page numbering hardcoded in English

### ISSUE THREAD

**Issue by Khoulaiz** [Sematre/typst-letter-pro#7]
1. Create a letter with more than 1 page

Expected: For a DIN Letter I would expect a German numbering scheme

Actual: Numbering is using "Page 1 of 2"

**Comment by Khoulaiz** [comment]
I know that I can change it. But why is English the default for a Letter following the German DIN standard? Everything else in this letter is in German. Sorry, but this doesn't make sense for me.

**Comment by Sematre** [comment]
The page numbering is the only place where in this library where there is a default English text. Everything else in this library is language independent.

Even tho this letter is following a German standard, [documents written in Typst default to the English language](https://typst.app/docs/reference/text/text/#parameters-lang). This is why I think it is the best choice to make English the default.

A good compromise could be to use the [linguify](https://typst.app/universe/package/linguify/) library that can derive the correct translation from the document context. Would this work for you?

**Comment by ondohotola** [comment]
> English is just the default. You can change it to German like this:
> 
> ```typst
> #show: letter-simple.with(
>   // [...]
>   page-numbering: (current-page, page-count) => [Seite #current-page von #page-count],
> )
> ```

Is there a way of testing for text.lang and then doping a conditional?

And, how to I change the size of the text `Page # of #`? I like it a little smaller than the text in the letter.

**Comment by Sematre** [comment]
> Is there a way of testing for text.lang and then doing a conditional?

You can get the current document language with this:
```typst
#context {
  let the_language = text.lang
  
  if the_language == "de" {
    "Deutscher Text!"
  } else {
    "English text!"
  }
}
```

> And, how to I change the size of the text `Page # of #`? I like it a little smaller than the text in the letter.

The `page-numbering` parameter expects you to return a `content` type, so you can just call e.g. `text(10pt)` to make it smaller.
```typst
#show: letter-simple.with(
  // [...]
  page-numbering: (current-page, page-count) => text(10pt)[Seite #current-page von #page-count],
)
```

**Comment by ondohotola** [comment]
Thanks

**Comment by ondohotola** [comment]
thank#.

I’ll use em though :-)-O

**Comment by Sematre** [comment]
This feature is now implemented and will be available in the upcoming `3.0` release.
